2022-08-monthly-report.org

```diff
@@ -0,0 +1,454 @@
+|-----------+------------+------------+-------|
+| Passing   | 6531       | 6740       | +209  |
+| Failed    | -          | -          | -     |
+| XFAIL     | 51         | 51         | -     |
+| XPASS     | -          | -          | -     |
+
+*** Bugs
+
+| Category    | Last Month | This Month | Delta |
+|-------------+------------+------------+-------|
+| TODO        |         55 |         53 |    -2 |
+| In Progress |         13 |         16 |    +3 |
+| Completed   |        178 |        200 |   +22 |
+
+*** Milestones Progress
+
+| Milestone                         | Last Month | This Month | Delta | Start Date     | Completion Date | Target        |
+|-----------------------------------+------------+------------+-------+----------------+-----------------+---------------|
+| Data Structures 1 - Core          |       100% |       100% | -     | 30th Nov 2020  | 27th Jan 2021   | 29th Jan 2021 |
+| Control Flow 1 - Core             |       100% |       100% | -     | 28th Jan 2021  | 10th Feb 2021   | 26th Feb 2021 |
+| Data Structures 2 - Generics      |       100% |       100% | -     | 11th Feb 2021  | 14th May 2021   | 28th May 2021 |
+| Data Structures 3 - Traits        |       100% |       100% | -     | 20th May 2021  | 17th Sept 2021  | 27th Aug 2021 |
+| Control Flow 2 - Pattern Matching |       100% |       100% | -     | 20th Sept 2021 | 9th Dec 2021    | 29th Nov 2021 |
+| Macros and cfg expansion          |       100% |       100% | -     | 1st Dec 2021   | 31st Mar 2022   | 28th Mar 2022 |
+| Imports and Visibility            |       100% |       100% | -     | 29th Mar 2022  | 13th Jul 2022   | 27th May 2022 |
+| Const Generics                    |        45% |        75% | +30%  | 30th May 2022  | -               | 17th Oct 2022 |
+| Intrinsics and builtins           |         0% |         0% | -     | 6th Sept 2022  | -               | 14th Nov 2022 |
+| Borrow checking                   |         0% |         0% | -     | TBD            | -               | TBD           |
+
+*** Risks
+
+| Risk                           | Impact (1-3) | Likelihood (0-10) | Risk (I * L) | Mitigation                                     |
+|--------------------------------+--------------+-------------------+--------------+------------------------------------------------|
+| Rust Language Changes          |            2 |                 7 |           14 | Target a specific Rustc version                |
+| Missing GCC 13 upstream window |            1 |                 6 |            6 | Merge in GCC 14 and be proactive about reviews |
+
+** Planned Activities
+
+- Closures
+- Bugs
+ 
+** Detailed changelog
+
+*** Libcore SIP Hasher
+
+We reached a milestone where we can fully compile one of our goal test cases libcore SIP Hasher taken from https://github.com/rust-lang/rust/blob/master/library/core/src/hash/sip.rs
+
+compiler explorer example: https://godbolt.org/z/bn4s54v67
+github-issue: https://github.com/Rust-GCC/gccrs/issues/1247
```

Comment:
The bug seems to be still in progress. Or maybe it wasn't updated.

Reply:
There are some subtasks there that need updated still so its not time to close the issue yet, but it does need an update.